hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -146,8 +146,8 @@ def __setattr__(self, name, value):
 
 
 class GenericStateMachine(
-        GenericStateMachineMeta('GenericStateMachine',
-                                (object,), {})  # type: ignore
+        GenericStateMachineMeta('GenericStateMachine', # type: ignore
```

Comment:
:man_facepalming: I need two spaces here!

Reply:
And only four at the start of the line, but Mypy doesn't care about either of those.